internal/appsec/dyngo/instrumentation/httpsec/http.go

```diff
@@ -39,6 +39,8 @@ type (
 	HandlerOperationRes struct {
 		// Status corresponds to the address `server.response.status`
 		Status int
+		// Headers corresponds to the address `server.response.headers`
+		Headers map[string][]string
```

Comment:
You can remove this new field for now because this feature is only about adding the response headers in the span. There are no security rules using this address so we can ignore it for now.

Reply:
Done, thanks.